doc/libghdl/Initialization.rst

```diff
@@ -0,0 +1,21 @@
+.. _LIB:Initialization:
+
+Initialization
+##############
+
+To be able to use `libghdl`, it first must be initialized.
+
+* Why does it must be initialized?
+* What does the initialization do?
+* Is there any use case where someone would want to use libghdl without initialization?
+* If no, then why it does not initialize itself automatically?
```

Comment:
One reason is because it is a low-level binding.  The other reason is because some initialization (like which ieee library is loaded) depends on some options.